src/Sentry/Internal/Http/HttpTransport.cs

```diff
@@ -259,6 +260,22 @@ await using (envelopeFile)
             }
         }
 
+        private string BuildAuthHeader(Dsn dsn)
+        {
+            var builder = new StringBuilder(100)
```

Comment:
When I tested it out (11 years ago :x) using a StringBuilder on a fixed-size small string similar to this case actually performed worse than a simple string concat.

Optimizing here (I'm guessing, to be sure we need some benchmarks) would be to have the fixed part assembled first, then just concatenate the timestamp to the end of the string:

```
           _authHeaderPrefix =
                $"Sentry sentry_version={_options.SentryVersion}," +
                $"sentry_client={_options.ClientVersion}," +
                $"sentry_key={dsn.PublicKey}," +
                (dsn.SecretKey is { } secretKey ? $"sentry_secret={secretKey}," : null) +
                "sentry_timestamp=";
           ...
           return _authHeaderPrefix + _clock.GetUtcNow().ToUnixTimeSeconds(); 
```

Reply:
Huh, interesting. I'll just switch it back, no need for premature optimization if it doesn't obviously provide obvious benefit 👍🏻